Sources/SwiftDocC/Indexing/RenderBlockContent+TextIndexing.swift

```diff
@@ -43,31 +47,41 @@ extension RenderBlockContent: TextIndexing {
         case .dictionaryExample(let e):
             return e.summary?.rawIndexableTextContent(references: references) ?? ""
         case .table(let t):
-            let content = t.rows.map {
-                return $0.cells.map {
-                    return $0.rawIndexableTextContent(references: references)
-                }.joined(separator: " ")
-            }.joined(separator: " ")
-            
+            let content = t.rows
+                .map {
+                    return $0.cells
+                        .map {
+                            return $0.rawIndexableTextContent(references: references)
+                        }
+                        .joined(separator: " ")
+                }
+                .joined(separator: " ")
```

Comment:
minor: This is fine but looks a bit strangely wrapped to me.

Reply:
I think this is the multiline dot expression setting.